packages/iov-keycontrol/src/keyring-entries/slip10.ts

```diff
@@ -76,6 +79,14 @@ export class Slip10KeyringEntry implements KeyringEntry {
     return new cls(JSON.stringify(data) as KeyringEntrySerializationString);
   }
 
+  private static readonly idsPrng: PseudoRandom.Engine = PseudoRandom.engines.mt19937().autoSeed();
+
+  private static generateId(): KeyringEntryId {
+    // this can be pseudo-random, just used for internal book-keeping
+    const code = PseudoRandom.string()(Slip10KeyringEntry.idsPrng, 16);
+    return `${code}` as KeyringEntryId;
```

Comment:
why ` ${}` ? this is different than implementation in ed25519.ts

Reply:
A leftover from before, will clean up